### ISSUE TITLE
Raise the limit of open files

### ISSUE THREAD

**Issue by Nicofuma** [rancher/os#359]
Hi,

Today I wanted to stop some containers before rebooting my server and I run into a strange issue: 
`Error response from daemon: Cannot stop foo-bar: [8] System error: open /proc/1053/stat: too many open files` after investigations I saw that the user docker daemon has 1023 opened file descriptors.

The first thing is I think there is a leak somewhere because I only have 18 running containers. (I saw a few of them in the docker repository but they are either marked as solved or seem unrelated). (**EDIT**: located in the rancher-agent, I'll open an issue).

The second thing is because rancher-os use docker for everything I think the limit should be raised to an higher number (even unlimited if possible).

**Comment by joshuacox** [comment]
in addition to the information in https://github.com/rancherio/rancher/issues/1219 I can say that I ran into the 'too many open files' problem on some generic ubuntu hosts as well that I had upgraded to use overlay doing something like this:
https://gist.github.com/joshuacox/25f7228a33e96f78d2a5
on a VM built using this packer build:
https://github.com/joshuacox/dacksible

**Comment by deniseschannon** [comment]
@Nicofuma  Can you confirm that you were also running Rancher agent on your hosts? I believe the leak is within rancher agent.

@ibuildthecloud Is it possible to raise the limit of number of open files in RancherOS?

**Comment by Nicofuma** [comment]
@deniseschannon I was running the agent yes and all the files were opened by it.

**Comment by deniseschannon** [comment]
Thanks @Nicofuma. I believe the too many open files error is a Rancher agent issue. 

I've renamed this issue as a request to raise the limit of open files in RancherOS.

**Comment by joshuacox** [comment]
as a fix I've been running something like:
```
docker rm `docker ps -a -q`
docker rmi -f `docker images -q`
```
and that usually clears the issue up, next time I run into this issue I'll be sure and get some lsof data

**Comment by imikushin** [comment]
This should be fixed by rancher/os-images#20
